compiler/transforms/convert_acc_to_csr.py

```diff
@@ -39,9 +42,14 @@ def get_acc(self, accelerator: StringAttr) -> acc.AcceleratorOp:
         acc_op = trait.lookup_symbol(self.module, accelerator)
         if not isinstance(acc_op, acc.AcceleratorOp):
             raise RuntimeError(
-                f"Invalid IR: no accelerator op for @{accelerator.data} found in module"
+                f"Invalid IR: Lowering acc2 for accelerator '{accelerator.data}'"
+                " requires an acc2.accelerator op to declare a symbol for "
+                f"@{accelerator.data}, but no such symbol was found"
+                " in the current module."
             )
-        return acc_op
+        # Use the retrieved acc_op to retrieve information from the registry
+        acc_info = get_registered_accelerators()[acc_op.name_prop.string_value()]
```

Comment:
Actually this is what i meant: otherwise it will be an ugly keyerror or something
```suggestion
        if acc_op.name_prop.string_value() not in get_registered_accelerators():
            raise RunTimeError(f"Unknown accelerator symbol {acc_op.name_prop.string_value()}")
        acc_info = get_registered_accelerators()[acc_op.name_prop.string_value()]
```